app/views/maintaining/contributor_teams/memberships/edit.html.haml

```diff
@@ -0,0 +1,15 @@
+#page-team-new
+  .sm-container.container
+    %header
+      %h1 Update #{@membership.user.github_username} contributor in team #{@team.name}
+      %hr.c-divider
+
+    %section.form-section
+      = form_for @membership, url: maintaining_contributor_team_contributor_team_membership_path(@team, @membership), as: :contributor_team_membership do |f|
```

Comment:
```suggestion
      = form_for @membership, url: maintaining_contributor_team_contributor_team_membership_path(@team, @membership) do |f|
```

I suspect you don't need this. It may change the param name in the controller, but I think it'll make it shorter, not longer.